### PR TITLE
Quickly written "How to" section

### LINKED ISSUE #8: Improve README with an HOW-TO section

### PR DESCRIPTION
**Pull request by petterik**
Fixes #8 

Not the best readme, but we can update as we change things.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,5 +6,36 @@ are not redirected but evaluated in the bridge as they require the JVM.
...
+echo '{:deps {org.clojure/clojure {:mvn/version "1.10.0"}
+              arcadia             {:local/root "../Arcadia"}}}' > deps.edn
+```
+
+2. Open and import the project with Cursive at that root.
+3. You should now have auto complete with the arcadia and clojure-clr sources.
+
+Now we'll want to talk to Arcadia via nREPL. To do this:
+
+1. Clone this repository: git@github.com:eponai/tools.cursive-arcadia-repl.git
```
Comment (dnegma) [inline README.md]: The URL is ~now~ not rendered properly, only the first part. Perhaps explicitly link this instead?

File: README.md [hunk 2]
```diff
@@ -6,5 +6,36 @@ are not redirected but evaluated in the bridge as they require the JVM.
...
+2. cd tools.cursive-arcadia-repl
```
Comment (dnegma) [inline README.md]: Want to highlight with `` that this is a command?

File: README.md [hunk 3]
```diff
@@ -6,5 +6,36 @@ are not redirected but evaluated in the bridge as they require the JVM.
...
+  ```sh
+cd Assets
+mkdir <project-name>
+cd <project-name>
+mkdir src test target
+echo '{:deps {org.clojure/clojure {:mvn/version "1.10.0"}
+              arcadia             {:local/root "../Arcadia"}}}' > deps.edn
+```
+
+2. Open and import the project with Cursive at that root.
```
Comment (dnegma) [inline README.md]: Unclear what this means, do you mean open Cursive? Import Arcadia? Isn't that already done in `deps.edn`?

File: README.md [hunk 4]
```diff
@@ -6,5 +6,36 @@ are not redirected but evaluated in the bridge as they require the JVM.
 [nREPL]: https://github.com/clojure/tools.nrepl
 [Arcadia Unity]: https://github.com/arcadia-unity/Arcadia
 
-## How to use
-TBD
+### How to use Cursive with Arcadia
```
Comment (dnegma) [inline README.md]: This section assumes we already have Clojure CLI, a Unity project, and Arcadia, so perhaps we can add a Prerequisites section with more details;
- Link to [Clojure Guide](https://clojure.org/guides/getting_started)
- Link to how to setup [Arcadia with Unity](https://github.com/arcadia-unity/Arcadia/wiki/Getting-Started)

File: README.md [hunk 5]
```diff
@@ -6,5 +6,36 @@ are not redirected but evaluated in the bridge as they require the JVM.
...
+
+In your Unity project with Arcadia, do:
+
+1. Create a new "regular" looking clojure project that has a deps.edn at its root. For example:
```
Comment (dnegma) [inline README.md]: Perhaps better to "decide" how to create the project, anyone already familiar with Clojure would probably have their own way of doing things anyway.
```suggestion
1. Create a new deps.edn clojure project. For example:
```

File: README.md [hunk 6]
```diff
@@ -6,5 +6,36 @@ are not redirected but evaluated in the bridge as they require the JVM.
...
+
```
Comment (dnegma) [inline README.md:16]: Perhaps we can explain what this is doing, and use proper syntax highlighting so it's a little clearer what is intended? E.g something like:

Create a `deps.edn` file with the following: 
```Clojure
{:deps {org.clojure/clojure {:mvn/version "1.10.0"}
        arcadia             {:local/root "../Arcadia"}}}
```

File: README.md [hunk 7]
```diff
@@ -6,5 +6,36 @@ are not redirected but evaluated in the bridge as they require the JVM.
...
+  ```sh
+cd Assets
+mkdir <project-name>
+cd <project-name>
+mkdir src test target
+echo '{:deps {org.clojure/clojure {:mvn/version "1.10.0"}
```
Comment (dnegma) [inline README.md]: Do we need to import this if we're working with Unity?

Comment (petterik) [reply README.md]: We still want a clojure version defined which Cursive thinks we're going to use. It'll use this clojure version to evaluate the code we write.

For reasons mentioned in https://github.com/eponai/tools.cursive-arcadia-repl/pull/5#discussion_r285393034 I still think we should **explicitly** define the clojure version used.

File: README.md (hunk 2 above)
Comment (petterik) [reply README.md]: I replaced this ordered list (1. 2. 3.) with a single code block. What do you think?

File: README.md (hunk 1 above)
Comment (petterik) [reply README.md]: https://github.com/eponai/tools.cursive-arcadia-repl/pull/6/files#r285394099

**Comment by petterik** [comment]
@dnegma Pushed new code. Changes:
* Added a prerequisites section.
* Re-arranged steps, so that one is going form "Unity specific" -> "Cursive specific"
* Added a testing section, to show how I have verified that the setup works.

File: README.md [hunk 8]
```diff
@@ -6,5 +6,70 @@ are not redirected but evaluated in the bridge as they require the JVM.
...
+4. Add the `Assets/Arcadia` directory to your `deps.edn` as a dependency:
+
+  ```clj
+{:deps {org.clojure/clojure {:mvn/version "1.10.0"}
+        arcadia             {:local/root "../Arcadia"}}}
+```
+
+5. In Intellij/Cursive, open and import the `Assets/<project-name>` directory.
+
+6. Create a [remote REPL in Cursive](https://cursive-ide.com/userguide/repl.html#Remote_REPLs) on port `7888`.
```
Comment (dnegma) [inline README.md]:
```suggestion
6. Create a [remote REPL in Cursive](https://cursive-ide.com/userguide/repl.html#remote-repls) on port `7888`.
```

File: README.md [hunk 9]
```diff
@@ -6,5 +6,70 @@ are not redirected but evaluated in the bridge as they require the JVM.
...
+```
+
+4. Add the `Assets/Arcadia` directory to your `deps.edn` as a dependency:
+
+  ```clj
+{:deps {org.clojure/clojure {:mvn/version "1.10.0"}
+        arcadia             {:local/root "../Arcadia"}}}
+```
+
+5. In Intellij/Cursive, open and import the `Assets/<project-name>` directory.
```
Comment (dnegma) [inline README.md]: Does this need to be imported as a Clojure Deps project in IntelliJ?

File: README.md (hunk 3 above)
Comment (petterik) [reply README.md]: This is outdated and has been reworded.

File: README.md (hunk 9 above)
Comment (petterik) [reply README.md]: Yes, or configured afterwards if one doesn't do it.

@dnegma Do you think this is too unclear or can we merge this PR without this change?

Comment (dnegma) [reply README.md]: We can merge this for now and add more changes later

**Review by dnegma — APPROVED** [review]

**Comment by dnegma** [comment]
@petterik reminder commit prefix `docs:`

**Pull request merged**